tools/stack_decode.py

```diff
@@ -30,7 +30,7 @@ def decode_stacktrace_log(input_source):
   backtrace_marker = "\[backtrace\] [^\s]+"
   trace_begin_re = re.compile(
       "^(.+)%s Backtrace obj<(.+)> thr<(\d+)" % backtrace_marker)
-  stackaddr_re = re.compile("%s thr<(\d+)> #\d+ (0x[0-9a-fA-F]+)$" % backtrace_marker)
+  stackaddr_re = re.compile("%s thr<(\d+)> #\d+ (0x[0-9a-fA-F]+) " % backtrace_marker)
```

Comment:
Nit: the space at the end is unnecessary here.

Reply:
I never trust regex to be greedy so I'm more comfortable leaving it in